spec/controllers/finders_controller_spec.rb

```diff
@@ -243,6 +235,23 @@
     end
   end
 
+  describe "popularity AB test B variant" do
```

Comment:
Could you also add a test for the A variant?

Reply:
Yep! I've added a test to the commit.